app/presenters/formats/local_transaction_presenter.rb

```diff
@@ -12,10 +12,6 @@ def details
       .merge(external_related_links: external_related_links)
     end
 
-    def registers_exact_route?
```

Comment:
Looks like this is being added in the previous comment and removed in this commit, so let's just remove completely from the diff